components/post_profile_picture/post_profile_picture.jsx

```diff
@@ -70,16 +79,21 @@ export default class PostProfilePicture extends React.PureComponent {
             user,
             isBot,
         } = this.props;
+
         const isSystemMessage = PostUtils.isSystemMessage(post);
         const fromWebhook = PostUtils.isFromWebhook(post);
+
         if (isSystemMessage && !compactDisplay && !fromWebhook && !isBot) {
             return <MattermostLogo className='icon'/>;
         }
-
         const fromAutoResponder = PostUtils.fromAutoResponder(post);
 
         const hasMention = !fromAutoResponder && !fromWebhook;
-        const src = this.getProfilePicSrcForPost(fromAutoResponder, fromWebhook);
+        const profileSrc = this.getProfilePicSrc(fromAutoResponder, fromWebhook);
```

Comment:
Looks like we don't accept `fromAutoResponder` nor `fromWebhook` in `getProfilePicSrc` -- can we remove?